### PR TITLE
people can now arm bears on castle station

### PR DESCRIPTION
**Pull request by ShiftyRail**
Follow-up PR to #28116

Not coded anywhere yet, waiting on #27884 to be resolved before to make conflict fixing easier
Sprites done by @beesil 

image

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Neat

**Comment by drawsstuff** [comment]
>circles wont work


good thing i got my anti bear underpants, right bartender?

**Comment by gurfan** [comment]
god bless

**Comment by gurfan** [comment]
Throwing shotguns at regular bears should make them into this, similarly to how throwing burger makes bears passive

**Comment by SonixApache** [comment]
bear cavalry when

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 1]
```diff
@@ -122,6 +122,30 @@
 /mob/living/simple_animal/hostile/bear/spare/GetAccess()
 	return get_all_accesses()
 
+/mob/living/simple_animal/hostile/bear/castle
+	name = "castle station space bear"
+	desc = "Drawing circles won't work this time!"
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/hostile/bear.dm:127]: Ironically, it probably will since you didn't override CanAttack

**Comment by Kurfursten** [comment]
Looks derelict, but feel free to reopen this if not

**Pull request closed without merging**